index.bs

```diff
@@ -205,9 +207,22 @@ ISSUE: Clarify the behavior of inline sessions
 
 Automatic composition {#automatic-composition}
 ---------
-<dfn>performs automatic composition</dfn>
 
-ISSUE: Add explanation here
+An [=xr device=] during an {{XRSessionMode/"immersive-ar"}} session will have its [=xr compositor=] <dfn>perform automatic composition</dfn> by drawing the rendered scene on top of the real-world background.
+
+In case the [=xr device=] is using a [=passthrough camera=], it MUST alpha-blend non-opaque rendered pixels on top of the [=passthrough camera=]'s [=aligned video stream=]. Such a device MUST have an {{XRSession/environmentBlendMode}} of {{alpha-blend}}.
+
+If the [=xr device=] is an additive light device where the real-world background is visible through the device's transparent display, it MUST ignore alpha values when performing automatic composition, instead, it MUST treat black pixels as transparent.
+
+Automatic composition MUST NOT automatically grant the page access to any additional information such as camera intrinsics, media streams, real-world geometry, etc.
+
+NOTE: It is not possible for the user to perform computer vision based on automatic composition. Future modules may enable such functionality.
+
+Automatic composition does not perform occlusion based on pixel depth relative to real-world geometry; rendered content will always be composed on top of the real-world background.
```

Comment:
If we are trying to constrain UAs here, should this be MUST NOT?  Performing such occlusion would be within the capability of today's AR headsets.

Reply:
>  Performing such occlusion would be within the capability of today's AR headsets.

Is it really? From the XR API the headsets only have the 2D scene views, they don't know the geometry of the rendered scene. We could add an API for this, of course, but it doesn't exist yet.

But yeah, this can be a MUST NOT.

Reply:
addressed.